extra/cockpit/foreman-cockpit-session

```diff
@@ -170,24 +170,32 @@ class Relay
   def initialize(proxy, params)
     @proxy = proxy
     @params = params
+    @inject_authorization = @params['ssh_user'] != 'root' && @params['effective_user_password']
   end
 
   def proxy_loop
     proxy1 = ProxyBuffer.new($stdin, @sock)
     proxy2 = ProxyBuffer.new(@sock, $stdout)
     proxy2.on_data do |data|
-      message = Cockpit.read_control(StringIO.new(data))
-      if message.is_a?(Hash) && message['command'] == 'authorize'
-        response = {
-          'command' => 'authorize',
-          'cookie' => message['cookie'],
-          'response' => @params['effective_user_password'],
-        }
-        proxy1.enqueue(Cockpit.encode_message(response))
-        ''
-      else
-        data
+      if @inject_authorization
+        sio = StringIO.new(data)
+        begin
+          message = Cockpit.read_control(sio)
+        rescue StandardError
+          # It is fine, carry on
```

Comment:
Why is it fine? :) (I mean it's better to leave it as a comment there)

Reply:
Added some sort of explanation there